docs/modules/reference/pages/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -235,33 +235,40 @@ Note that you must include the `monitor` section for each service in your defini
   <parameter key="ds-name" value="opennmslogin"/>
   <parameter key="page-sequence">
     <page-sequence>
-      <page path="/opennms/login.jsp"
-            port="8980"
-            successMatch="Password" />
-      <page path="/opennms/j_spring_security_check"
-            port="8980"
-            method="POST">
+      <page path="/opennms/login.jsp" <1>
+            port="8980" <2>
+            successMatch="Password" /> <3>
+      <page path="/opennms/j_spring_security_check" <1>
+            port="8980" <2>
+            method="POST"> <4>
         <parameter key="j_username" value="admin"/>
         <parameter key="j_password" value="admin"/>
         <header name="foo" value="bar"/>
       </page>
-      <page path="/opennms/index.jsp"
-            port="8980"
-            successMatch="Log Out" />
-      <page path="/opennms/event/index"
-            port="8980" successMatch="Event Queries" />
-      <page path="/opennms/j_spring_security_logout"
-            port="8980"
-            method="POST"
-            response-range="300-399"
-            locationMatch="/opennms" />
+      <page path="/opennms/index.jsp" <1>
+            port="8980" <2>
+            successMatch="Log Out" /> <3>
+      <page path="/opennms/event/index" <1>
+            port="8980" <2>
+            successMatch="Event Queries" /> <3>
+      <page path="/opennms/j_spring_security_logout" <1>
+            port="8980" <2>
+            method="POST" <4>
+            response-range="300-399" <5>
+            locationMatch="/opennms" /> <6>
     </page-sequence>
   </parameter>
 </service>
 
-<monitor service="OpenNMS-Web-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <1>
+<monitor service="OpenNMS-Web-Login" class-name="org.opennms.netmgt.poller.monitors.PageSequenceMonitor"/> <7>
 ----
-<1> Required *monitor* section.
+<1> The relative URL to call in the request.
+<2> Host port number to use when connecting.
+<3> Text to look for in the response body.
+<4> HTTP method (GET or POST).
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.